src/components/AccountManager/AccountManager.js

```diff
@@ -0,0 +1,253 @@
+        getUserInfo();
+
+        if(resetPasswordOne === "" || resetPasswordTwo === ""){
+            setPasswordsMatch("Enter your new password...");
+        }
+        else if(resetPasswordOne === resetPasswordTwo){
+            setPasswordsMatch("Passwords Match!");
+        }
+        else {setPasswordsMatch("Passwords do not match!")}
+
+    }, [updateInfo, resetPasswordOne, resetPasswordTwo]);
+
+    useEffect(() => {
+        if(!isGuest){
+            console.log("NotAGuest"+isGuest.toString());
+            setConfirmationText("Confirm");
+        }
+        else{
+            console.log("IsAGuest"+isGuest.toString());
+            setConfirmationText("DISABLED");
+        }
+    }, [isGuest]);
+
+    const changeInfo = async () => {
+        if(!isGuest){
+            let msg = await userInfo.updateOne("users", {userId: context.mongoHook.authenticatedUser.id},
+                {$set: {email: changeEmail, first_name: changeFirstName, last_name: changeLastName}});
+            setUpdateInfo(!updateInfo);
+            switch(msg.matchedCount){
+                case 0: msg = "Matched: 0\nAccount not found!";
+                    break;
+                case 1: msg = "Matched: 1\nAccount found!\n"+context.mongoHook.authenticatedUser.id;
+            }
+            console.log(msg);
+        }
+    };
+
+    const deleteAccount = () => {
+        setVisibleDeleteConfirmation(!visibleDeleteConfirmation);
+    };
+
+    const deleteAccountConfirmation = () => {
+        alert("Account Deleted! I guess....\nBye Bye =,(")
+    };
+
+    const [visibleDeleteConfirmation, setVisibleDeleteConfirmation] = useState(false);
+    const onDismissDeleteConfirmation = () => setVisibleDeleteConfirmation(false);
+
+    //Notes
+    /*
+        User pool pass and descriptor pass are separate from eachother.
+        Modification to both use pool pass and database plain text are required until changed!
+
+        For the delete feature it will need to delete the users/descriptors value of itself, and the user
+        from the user pool
+     */
+
+    return (
+        <div>
+            <label>UserID: {context.mongoHook.authenticatedUser.id}</label>
+            <br />
+            <label>Name: {firstName} {lastName}</label>
+            <br />
+            <label>Email: {email}</label>
+            <br />
+            <label>(DEV USE ONLY) Password: {password}</label>
+
+            <br />
+            <br />
+
+            <Button color="primary" onClick={toggleInfoSetup} style={{ marginBottom: '1rem' }}>Edit Personal Info</Button>
+            <label>--</label>
+            <Button color="primary" onClick={toggleResetPassword} style={{ marginBottom: '1rem' }}>Change Password</Button>
+            <Collapse isOpen={isInfoSetupOpen}>
+                <label>WIP - New Info</label>
+                <InputGroup className={"inputGroupLogin"}>
+                    <InputGroupAddon className={"pre-pend"} addonType="prepend">FirstName</InputGroupAddon>
```

Comment:
Great use of custom css classes!! Just a note, you don't have to put the strings in curly braces ( I know webstorm auto fills the curly braces), just a coding flavor preference though :)